template/private-npm-package/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "<%- fullModuleName %>",
+  "description": "<%- description %>",
+  "version": "0.0.0-semantically-released",
+  "main": "lib-commonjs",
+  "module": "lib-es2015",
+  "typings": "lib-types",
+  "files": [
+    "lib"
+  ],
+  "sideEffects": false,
+  "license": "UNLICENSED",
+  "scripts": {
+    "build": "skuba build",
```

Comment:
Should this be:

```suggestion
    "build": "skuba build-package",
```

Reply:
would the `prerelease` also be `skuba build-package` ?

Reply:
Ah good call! I'd like to get rid of pre- and post-scripts where we can, as they are unnecessary indirection and [not supported in Yarn 2](https://yarnpkg.com/advanced/lifecycle-scripts).

I'd steal from Koala here: https://github.com/seek-oss/koala/blob/master/package.json#L43